src/init.js

```diff
@@ -12,30 +12,44 @@ export default function init(data) {
         else d.measure_unit = d[this.config.measure_col];
     });
 
-    const sortedData = data.sort((a, b) => {
-        const aValue = a.measure_unit,
-            bValue = b.measure_unit,
-            leftSort = aValue < bValue,
-            rightSort = aValue > bValue,
-            aID = a[this.config.id_col],
-            bID = b[this.config.id_col],
-            aTime = a[this.config.time_col],
-            bTime = b[this.config.time_col];
+    const sortedData = data
+        .filter(d => /^[0-9\.]+$/.test(d[this.config.value_col]))
```

Comment:
This is just dropping rows where the value column is non-numeric? 

Probably want to do this in a more robust way, and throw some errors/warnings when values are dropped. I'll put in an issue for future development.

Reply:
See #25